torch/nn/quantized/modules/conv.py

```diff
@@ -174,6 +174,16 @@ def __setstate__(self, state):
         self.zero_point = state[13]
         self.training = state[14]
 
+    def __deepcopy__(self, memo):
+        new_instance = type(self).__new__(type(self))
+        torch.nn.Module.__init__(new_instance)
```

Comment:
just so I understand the fix correctly, this is the line which was missing in the default `__deepcopy__` implementation, right?

Reply:
Yeah

cc @gmagogsfm @SplitInfinity can we confirm that the `__getstate__`/`__setstate__` protocol in TorchScript aligns with Python? It seems strange to have to reconcile behavior like this

Reply:
Hi James,

Could you clarify why this is related TorchScript protocol? Is quantization module somehow scripted? I didn't find code to script them.

Additionally, what do you think about changing to fix from overriding `deepcopy` to making `getstate` and `setstate` call their base class methods? `nn.Module`'s `__setstate__` has logic to handle some absent members, and need to be called when overriding `__{g,s}etstate__`.

See this example:

```
class CustomModule(torch.nn.Module):
    def __init__(self, v):
        super().__init__()
        self.v = v

    def __setstate__(self, state):
        super().__setstate__(state)          # Commenting out this line would cause `dir()` to fail on module due to missing members
        self.v = state['v']

n = CustomModule(1)
print(n.__dict__)
print(dir(n))
copied = copy.deepcopy(n)
print(copied.__dict__)
print(dir(copied))
```